hack/make/run

```diff
@@ -36,15 +41,14 @@ if [ -n "$DOCKER_EXPERIMENTAL" ]; then
 	extra_params="$extra_params --experimental"
 fi
 
-dockerd="dockerd"
+dockerd="$(command -v dockerd)"
```

Comment:
I will move `dockerd_command` just after the `if` block in which the presence of dockerd is checked. I think it's more suitable place, since the `command -v docker` is called literally few lines before as a check. Then the `dockerd` variable can be created just before potentially being changed by presence of `DELVE_PORT` or `DOCKER_ROOTLESS`.

```bash
if ! command -v dockerd &> /dev/null; then
	echo >&2 'error: binary-daemon or dynbinary-daemon must be run before run'
	false
fi

DOCKER_COMMAND="$(command -v dockerd)"

# ...

dockerd=( "$DOCKER_COMMAND" )

if [ -n "$DELVE_PORT" ]; then
	dockerd=(
		dlv
		--listen="0.0.0.0:$delve_listen_port"
		--headless=true
		--log
		--api-version=2
		--only-same-user=false
		--check-go-version=false
		--accept-multiclient
		exec "${dockerd[@]}" --
	)
fi

```